### PR TITLE
Add documentation for missing qt bindings in unreal

### PR DESCRIPTION
**Pull request by antirotor**
## Changelog Description
Adding information about missing Qt bindings in Unreal and how to deal with them.

### CODE REVIEW

**Review by moonyuet — APPROVED** [review]

File: website/docs/addon_unreal_admin.md [hunk 1]
```diff
@@ -37,3 +37,28 @@ To use a custom path for the compiled plugin, you need to set an environment var
...
+
+```python
+import pip
+pip.main(["install", "pyside6"])
+```
+
+Be aware that calling **pip** like so is deprecated by pip itself and might not work in newer versions.
+
+:::note
+For versions of Unreal Engine >= 5.4, use `pyside6`, use `pyside2` for the lower versions.
```
Comment (MustafaJafar) [inline website/docs/addon_unreal_admin.md:63]: could we rephrase it ? 
Maybe something like this: 
```suggestion
For Unreal Engine versions `>= 5.4`, use `pyside6`. For earlier versions, use `pyside2`.
```

File: website/docs/addon_unreal_artist.md [hunk 2]
```diff
@@ -28,7 +28,18 @@ import loaderVideoUE from './assets/video/unreal_launch.mp4'
 
 ### Prerequisites
 
-AYON supports Unreal Engine 5.0 and 5.1. To use it, you need to install the [AYON Integration plugin](https://temp.com) from the Unreal Marketplace.
+AYON supports the latest Unreal Engine versions. To use it, you need to install the [AYON Integration plugin](https://www.unrealengine.com/marketplace/en-US/product/ayon-pipeline-integration) from the Unreal Marketplace.
+
+:::note
+AYON data folders are color coded by unique color scheme for easier navigation. As seen on the picture above.
```
Comment (MustafaJafar) [inline website/docs/addon_unreal_artist.md:34]: > As seen on the picture above.

Is there a picture above ? I couldn't find.

Comment (antirotor) [reply website/docs/addon_unreal_artist.md:34]: not sure, I wasn't touching this, just fixing the URL for Integration plugin

**Review by MustafaJafar — APPROVED** [review]
Looks Good!

**Pull request merged**